snomed/com.b2international.snowowl.snomed.core.rest.tests/src/com/b2international/snowowl/snomed/core/rest/components/SnomedDescriptionApiTest.java

```diff
@@ -313,6 +314,28 @@ public void testReactivateThenInactivateDescription() throws Exception {
 			.body("members.items.active", not(hasItem(true)))
 			.body("members.items.effectiveTime", not(hasItem(not(equalTo(effectiveDate)))));
 	}
+	
+	@Test
+	public void testSelectiveAcceptabilityInactivation() throws Exception {
+		String descriptionId = createNewDescription(branchPath, 
+				Concepts.ROOT_CONCEPT, 
+				Concepts.SYNONYM, 
+				ImmutableMap.of(
+						Concepts.REFSET_LANGUAGE_TYPE_UK, Acceptability.ACCEPTABLE,
+						Concepts.REFSET_LANGUAGE_TYPE_US, Acceptability.ACCEPTABLE)
+				);
+		
+		Json inactivateRequestBody = Json.object(
+			"active", false,
+			"acceptability", Json.object(Concepts.REFSET_LANGUAGE_TYPE_US, Acceptability.ACCEPTABLE),
+			"commitComment", "Inactivate released description"
+		);
+
+		updateComponent(branchPath, SnomedComponentType.DESCRIPTION, descriptionId, inactivateRequestBody).statusCode(204);		
+		SnomedDescription description = getComponent(branchPath, SnomedComponentType.DESCRIPTION, descriptionId, "members()")
+			.extract().as(SnomedDescription.class);
+		assertTrue(description.getAcceptabilityMap().containsKey(Concepts.REFSET_LANGUAGE_TYPE_US));
```

Comment:
I would also validate that the description is indeed inactive, plus the UK language refset member **is inactive** and the US language refset member **is still active**, otherwise this test does not cover the code changes in the change processor class.

Reply:
Well, it is sort of covered, but the status of the language refset members is the key point here